src/com/owncloud/android/db/UploadDbObject.java

```diff
@@ -0,0 +1,321 @@
+package com.owncloud.android.db;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+import android.accounts.Account;
+import android.content.Context;
+import android.util.Base64;
+
+import com.owncloud.android.authentication.AccountUtils;
+import com.owncloud.android.datamodel.OCFile;
+import com.owncloud.android.db.UploadDbHandler.UploadStatus;
+import com.owncloud.android.files.services.FileUploadService.LocalBehaviour;
+import com.owncloud.android.lib.common.operations.RemoteOperationResult;
+import com.owncloud.android.lib.common.utils.Log_OC;
+
+/**
+ * Stores all information in order to start upload operations. PersistentUploadObject can
+ * be stored persistently by {@link UploadDbHandler}.
+ * 
+ * @author LukeOwncloud
+ * 
+ */
+public class UploadDbObject implements Serializable {
+
+    /** Generated - should be refreshed every time the class changes!! */
+    ;
+    private static final long serialVersionUID = -2306246191385279928L;
+
+    private static final String TAG = "UploadDbObject";
```

Comment:
We defined the TAG string for log messages as:
className.class.getSimpleName.

Could you define it in this way? For this class TAG = UploadDbObject.class.getSimpleName()